### PR TITLE
adds manual DNS config to 4.12 rel notes

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to 4.12 only
- [Jira](https://issues.redhat.com/browse/CFE-606)
- [Preview](https://51111--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-networking)
- @thejasn @ShudiLi   ptal

### CODE REVIEW

**Comment by thejasn** [comment]
LGTM
Awaiting docs [PR](https://github.com/openshift/openshift-docs/pull/49752) to merge for `lgtm` label

**Comment by Amrita42** [comment]
@thejasn  the doc PR is now merged fyi.

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -86,6 +86,13 @@ You can now use the {product-title} Custom Metrics Autoscaler Operator to dynami
 
 For more information, see xref:../networking/ingress-operator.adoc#nw-autoscaling-ingress-controller[Autoscaling an Ingress Controller].
 
+[id="ocp-4-12-nw-configure-dns-management"]
+==== Configuration of an Ingress Controller for manual DNS management
+
+You can now configure an Ingress Controller to stop automatic DNS management and start manual DNS management. Set `dnsManagementPolicy` to specify when it should be automatically or manually managed.
```
Comment (kalexand-rh) [inline release_notes/ocp-4-12-release-notes.adoc:92]: > ! should verb
Whenever possible, rewrite in a more direct way. Do not use "should" to refer to an action that must be performed.

> Generally, when you specify the name of a command, parameter, or option in text, specify the word command, parameter, or option after the name.
Command, parameter, and option names are not translated; without the identifying noun, users might be confused about what the name represents.

**Review by kalexand-rh — COMMENTED** [review]
Has this been approved by QE?

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
Comment (Amrita42) [reply release_notes/ocp-4-12-release-notes.adoc:92]: @kalexand-rh The main feature/doc PR has been approved by QE , the same addition to the rel notes, is QE ack required?

**Comment by Amrita42** [comment]
Closing this PR due to merge conflicts , created a new PR taking fwd all feedback >> https://github.com/openshift/openshift-docs/pull/51417

**Pull request closed without merging**